src/elm/Page/PublicProfile.elm

```diff
@@ -49,7 +49,7 @@ type alias Model =
 type Status
     = Loading
     | LoadingFailed (Graphql.Http.Error (Maybe Profile))
-    | Loaded Profile
```

Comment:
Why not just have a `NotFound` value for `Status` type? This way we don't have to rely on maybe. Types > Maybe almost all the times